frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -454,6 +467,22 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
         ],
     }),
     listeners(({ props, values, actions, cache }) => ({
+        playerErrorSeen: ({ errorEvent }) => {
+            const fingerprint = encodeURIComponent(
+                errorEvent.message + errorEvent.filename + errorEvent.lineno + errorEvent.colno
+            )
+            if (values.reportedReplayerErrors.has(fingerprint)) {
+                return
+            }
+            captureException(errorEvent.error, {
+                extra: { fingerprint },
+                tags: { recordingId: values.sessionRecordingId },
```

Comment:
`sessionRecordingId` probably has very high cardinality, usually, tags are used as "ReplayFeature" instead of Ids, would you query errors per `sessionRecordingId` in Sentry? or just errors per feature.

Reply:
🤯